pennylane/beta/plugins/default_tensor.py

```diff
@@ -490,10 +554,10 @@ def expval(self, observable, wires, par):
         tensors = []
         for o, p, w in zip(observable, par, wires):
             A = self._get_operator_matrix(o, p)
```

Comment:
Does the `_get_operator_matrix` method take into account tensor products of observables smartly, or is it still doing `np.kron()` under the hood?

Reply:
It pulls directly the individual tensors from `_operation_map` and `_observable_map`, then creates a node for each independent observable. I don't see any place where `np.kron()` is being used